### PR TITLE
Update proposal for donate.md

### PR DESCRIPTION
**Pull request by TernundaHunter**
I've been thinking about this page for awhile since I am constantly advertising for Patron in the server. Here's a friendly recommendation for the donation page. The $400 amount number is just an assumption based on the amount of donations coming in on Patreon at the moment.

### CODE REVIEW

File: public/markdown/donate.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
 # Help support FredBoat♪♪
-FredBoat♪♪ has constantly been growing at a ridiculous rate ever since it launched. Building a bot of this scale has proven to be much more challenging than coding a simple music bot in the first place. Though by far the biggest challenge of this project is finding the money to provide enough server power.
+We love bringing music right to your Discord Server through FredBoat♪♪. Ever since the launch of FredBoat♪♪, it has grown in popularity at an incredible rate. Improvements are constantly being made to the bot, but the largest challenge is providing enough server power. We have over 450k FredBoat♪♪ bots floating out there in the Discord ocean!
```
Comment (Frederikam) [inline public/markdown/donate.md]: This is misleading. We don't have more than a handful of instances of the bot, and we only have 400k servers

File: public/markdown/donate.md [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 
-It currently costs __over $200 per month__ just to rent servers, but even this will not be enough to sustain the tremendous growth and quality is also starting to suffer during the weekends.
+It currently costs __over $400 a month__ in server rent for the sheer amount of power and data we need to keep FredBoat♪♪ going. However, this isn't even enough to keep quality from suffering during weekend peak times.
```
Comment (Frederikam) [inline public/markdown/donate.md]: I don't fork over that much money yet, but we will be there once we get something sorted out with my provider

File: public/markdown/donate.md [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
+
+You can support the development of FredBoat♪♪ while also enjoying some exclusive features we are unable to impliment in the free bot.
 
 ## Perks of donating
-As a patron of $2 or more per month you will gain access to FredBoat Patron in one server of your choosing. This comes with a few extra perks that we can't provide to the general public:
+As a patron of $2 or more per month you will gain access to FredBoat Patron in one server of your choosing which includes every feature you already enjoy in the free bot as well as:
```
Comment (Frederikam) [inline public/markdown/donate.md]: I intentionally left a period to not have the sentence become this long

File: public/markdown/donate.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
 
-It currently costs __over $400 a month__ in server rent for the sheer amount of power and data we need to keep FredBoat♪♪ going. However, this isn't even enough to keep quality from suffering during weekend peak times.
+It currently costs __over $200 a month__ in server rent for the sheer amount of power and data we need to keep FredBoat♪♪ going. However, this isn't even enough to keep quality from suffering during weekend peak times.
 
 You can support the development of FredBoat♪♪ while also enjoying some exclusive features we are unable to impliment in the free bot.
 
 ## Perks of donating
-As a patron of $2 or more per month you will gain access to FredBoat Patron in one server of your choosing which includes every feature you already enjoy in the free bot as well as:
+As a patron of $2 or more per month you will gain access to FredBoat Patron in one server of your choosing as well as:
```
Comment (Frederikam) [inline public/markdown/donate.md]: Still a bit misleading. You're making it sounds like FredBoat Patron and the perks of having the bot are separate.

Comment (TernundaHunter) [reply public/markdown/donate.md]: The sentence before ## Perks of Donating is simply a lead in, a transitional statement to encourage the reader to continue. It states that, by supporting the bot, they will be able to enjoy features exclusive to them as a patron. How would you recommend it be changed if this is not clear enough?

File: public/markdown/donate.md (hunk 2 above)
Comment (napstr) [reply public/markdown/donate.md]: I dont see a reason to be disclosing this amount on the donate page. Its something that needs constantly updates, and also feels overall like a weird thing to disclose in the first place.

Comment (TernundaHunter) [reply public/markdown/donate.md]: I can understand why it was originally offered. More often than not, people feel the need to prove they actually need the money for x, y, or z reason. However, the beautiful thing about Fredboat Patron is you're literally providing a service for their money. I would almost say "Fredboat Premium," but I don't want to really recommend drastically changing the flavor of paying. People like to feel good about supporting something they appreciate and believe in.

File: public/markdown/donate.md [hunk 5]
```diff
@@ -1,22 +1,23 @@
 # Help support FredBoat♪♪
-FredBoat♪♪ has constantly been growing at a ridiculous rate ever since it launched. Building a bot of this scale has proven to be much more challenging than coding a simple music bot in the first place. Though by far the biggest challenge of this project is finding the money to provide enough server power.
+We love bringing music right to your Discord Server through FredBoat♪♪. Ever since the launch of FredBoat♪♪, it has grown in popularity at an incredible rate. Improvements are constantly being made to the bot, but the largest challenge is providing enough server power. We have over 400k FredBoat♪♪ bots floating out there in the Discord ocean and, often, 2k+ deployed at any given time!
```
Comment (napstr) [inline public/markdown/donate.md]: the 400k will need an update real soon :upside_down_face:

**Pull request merged**